assembl/graphql/schema.py

```diff
@@ -326,6 +326,31 @@ class Meta:
         interfaces = (Node, )
         only_fields = ('id', 'body', 'important')
 
+class LocalePreference(graphene.ObjectType):
+    locale = graphene.String()
+    name = graphene.String(in_locale=graphene.String(required=True))
+
+    def resolve_name(self, args, context, info):
+        in_locale = args.get('in_locale') or None
+        locale_model = models.Locale.get_or_create(in_locale)
+
+        name = models.LocaleLabel.names_of_locales_in_locale([self.locale],
+                                                             locale_model)
+        if not name:
+            # If the locale label does not exist, fallback on English
+            locale_model = models.Locale.get_or_create('en')
+            name = models.LocaleLabel.names_of_locales_in_locale([self.locale],
+                                                                 locale_model)
+            return name['en']
+
+        return name[self.locale]
+
+
+class DiscussionPreference(graphene.ObjectType):
```

Comment:
maybe rename to DiscussionPreferences? (with an s)

Reply:
Okay, done :)